coriolisclient/cli/migrations.py

```diff
@@ -141,6 +143,10 @@ def get_parser(self, prog_name):
                             dest="instances",
                             help='An instances to be migrated, can be '
                             'specified multiple times')
+        parser.add_argument('--network-map', required=True,
+                            dest="network_map",
+                            help='JSON encoded data mapping from '
```

Comment:
This is phrased rather confusingly, please change it to something like:
`JSON mapping between identifiers of networks on the source and identifiers of networks on the destination.`